### ISSUE TITLE
onnx ，

### ISSUE THREAD

**Issue by picEmily** [ultralytics/yolov5#664]
## ❔Question
I get onnx model by onnx_export.py with yolov5s.pt, the layers are 140.
_**Fusing layers...
Model Summary: 140 layers, 7.45958e+06 parameters, 7.45958e+06 gradients**_

I get onnx model by onnx_export.py with my trained model, the layers are 120.
_**Fusing layers...
Model Summary: 120 layers, 6.8612e+06 parameters, 6.31725e+06 gradients**_

Why?

**Comment by Aktcob** [comment]
convolution and batch norm confusion
